rbc/tests/test_omnisci_column_default.py

```diff
@@ -83,7 +83,7 @@ def test_default_sizer(omnisci, suffix, kind, expected):
     query = f"select * from table({fn}("
     for i, n in enumerate(kind):
         cols = codes[n]
-        if str.isdigit(n):
+        if n.isdigit:
```

Comment:
`isdigit` is a method, so use `n.isdigit()`.

I wonder if the tests fail when using the master branch of omniscidb-internal?